airbyte-webapp/src/pages/OnboardingPage/components/useGetStepsConfig.tsx

```diff
@@ -39,29 +39,32 @@ const UseGetStepsConfig = (
     () => [
       {
         id: StepsTypes.CREATE_SOURCE,
-        name: <FormattedMessage id="onboarding.createSource" />,
-        onSelect: hasSources
-          ? () => updateStep(StepsTypes.CREATE_SOURCE)
-          : undefined
+        name: <FormattedMessage id="onboarding.createSource" />
+        // don't navigate by steps for now
```

Comment:
i'm fine with this. why did you choose this? just curious. it seems like this feature has been a little finicky.

Reply:
Yeah, we discussed it during the call, that we need to disable those button. Currently this functionality is a bit broken, should we open 1 more task to fix it? As we have changed check connection functionality - it is now much more easier to support it.